core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.component.ts

```diff
@@ -223,7 +223,7 @@ export class TranslationSuggestionReviewModalComponent implements OnInit {
     }
     this.activeSuggestion = this.activeContribution.suggestion;
     this.contextService.setCustomEntityContext(
-      AppConstants.IMAGE_CONTEXT.EXPLORATION_SUGGESTIONS,
+      AppConstants.ENTITY_TYPE.EXPLORATION,
```

Comment:
Can you add a comment here explaining why we are setting context like this?

Reply:
@Ash-2k3 What about when we set the entity context here? https://github.com/oppia/oppia/blob/develop/core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts#L401? Also, it's not really clear what "mismatch of contexts" means and how that affects the images. Can you explain in the comment?